stock_deposit/data/stock_deposit_data.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- © 2016 Sergio Teruel <sergio.teruel@tecnativa.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+<data noupdate="1">
+
+    <record id="stock_deposit_regularize_sequence" model="ir.sequence">
```

Comment:
Why this sequence? It should use the delivery order picking type sequence, or you will find under the same card pickings with different sequences. Or do you prefer this way?